drivers/CMakeLists.txt

```diff
@@ -26,6 +26,7 @@ add_subdirectory_if_kconfig(spi)
 add_subdirectory_if_kconfig(usb)
 add_subdirectory_if_kconfig(watchdog)
 add_subdirectory_if_kconfig(wifi)
+add_subdirectory_if_kconfig(can)
```

Comment:
That should go in the previous patch, the one adding the Kconfig infrastructure.